codegen/pom.xml

```diff
@@ -143,6 +138,14 @@
             <groupId>com.fasterxml.jackson.jr</groupId>
             <artifactId>jackson-jr-stree</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
```

Comment:
The code generator was already using this. Because it was removed from the aws-json-protocol package, we have to re-add it here. It's fine if the code generator uses Jackson, because it's just a build-time dependency.